src/Streamly/Internal/Data/Stream/Serial.hs

```diff
@@ -1,5 +1,5 @@
 {-# LANGUAGE UndecidableInstances #-}
-{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
Why did this have to change?

Reply:
we are using deprecated type for backward compatibility.
Want to do minimal change in *Depricated Module*.